app/controllers/editions_controller.rb

```diff
@@ -27,6 +30,23 @@ def show
   alias_method :metadata, :show
   alias_method :unpublish, :show
 
+  def update
+    @resource.assign_attributes(permitted_params)
+
+    if @resource.save
+      UpdateWorker.perform_async(resource.id.to_s)
+      flash.now[:success] = "Edition updated successfully."
+    else
+      @resource = resource
```

Comment:
I know it was in the legacy controller, but I'm not sure this is actually doing anything/is needed—I tried commenting it out and it still seems to work. Thoughts?

Reply:
Yes, makes sense, have deleted that line.